app/assets/stylesheets/components/_space-misc.scss

```diff
@@ -18,9 +18,13 @@
 .mtn1 { margin-top: -$space-1; }
 .mtn2 { margin-top: -$space-2; }
 .mtn3 { margin-top: -$space-3; }
+.mtn4 { margin-top: -$space-5; }
 
 @media #{$breakpoint-sm} {
   .sm-mr-20p { margin-right: 20px; }
+  .sm-mt0 { margin-top: 0; }
+  .sm-mbn3 { margin-bottom: -$space-3; }
+  .sm-mtn1 { margin-top: -$space-1; }
```

Comment:
Looks like some of those were removed from the partial in your PR. I'll go through and check the rest of the classes that I added here